lfucg-exactions/server/accounts/serializers.py

```diff
@@ -22,9 +23,23 @@ class AccountSerializer(serializers.ModelSerializer):
 
     address_state_display = serializers.SerializerMethodField(read_only=True)
 
+    balance = serializers.SerializerMethodField(read_only=True)
+    credit_availability = serializers.SerializerMethodField(read_only=True)
+
     def get_address_state_display(self, obj):
         return obj.get_address_state_display()
 
+    def get_balance(self, obj):
```

Comment:
When Account is serialized, calculate_account_balance is called once for each field. See options in the link below to only call it once.
https://stackoverflow.com/questions/40663579/call-method-once-to-set-multiple-fields-in-django-rest-framework-serializer